app/assets/scripts/components/dashboard/node-time-series-card.js

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react';
+import { PropTypes as T } from 'prop-types';
+import styled from 'styled-components';
+import ErrorMessage from '../error-message';
+import Card, {
+  CardHeader as BaseHeader,
+  CardTitle,
+  CardHeadline,
+} from '../card';
+import TabbedSelector from '../tabbed-selector';
+import LineChart from '../line-chart';
+import InfoButton from '../info-button';
+
+const ChartContainer = styled.div`
+  max-height: 24rem;
+  min-height: 20rem;
+  padding: 0;
+`;
+
+const CardHeader = styled(BaseHeader)`
+  display: grid;
+  grid-template-rows: min-content 1fr;
+  grid-gap: 0.5rem;
+`;
+
+export default function TimeSeriesCard({ parameters, data, titleInfo }) {
```

Comment:
Why did you duplicate the component? We can reuse the card and chart component for both time series data sources, fetched and generated

Reply:
There is definitely a cleaner way of handling this, but the timeseries accesses the /averages endpoint with either a project id or a single location. The response for passing in multiple locations as a group is different than the prior cases and needs to different logic for the card and chart to consume it properly. 

Since I am already hitting the /averages endpoint for the nodes-dashboard, there is no need to make that request again in the timeseries. For simplicity's sake, I just created a different timeseries comp that consumes the data that was already fetched in the nodes-deashboard and toggles the results in the tab